website/client/js/controllers/inviteToGroupCtrl.js

```diff
@@ -44,8 +44,13 @@ habitrpg.controller('InviteToGroupCtrl', ['$scope', '$rootScope', 'User', 'Group
 
       Groups.Group.invite($scope.group._id, invitationDetails)
         .then(function() {
-          Notification.text(window.env.t('invitationsSent'));
-          _resetInvitees();
+           if ((inviteMethod === 'email' && invitationDetails.emails.length > 1) || (inviteMethod === 'uuid' && invitationDetails.uuids.length > 1)) { // test if it's inviting more than 1 emails or uuid
```

Comment:
> `inviteMethod === 'email'` then `invitationDetails.uuids` is not supposed to exist

What about just checking if either exists then? 
```javascript
if (invitationDetails.emails || invitationDetails.uuids) {
```

Or for slightly better readability without changing your logic, you could do something like

```javascript
var isInvitedByEmail = inviteMethod === 'email' && invitationDetails.emails.length > 1;
var isInvitedByUUID = inviteMethod === 'uuid' && invitationDetails.uuids.length > 1;

if (isInvitedByEmail || isInvitedByUUID) {
// etc.
}
```